themes/docsy/layouts/videos/single.html

```diff
@@ -23,6 +23,13 @@ <h2 class="mb-4"> {{- .Title -}}</h2>
             <p>{{ .Description }}</p>
         </div>
     </div>
+    <div class="row justify-content-center mb-5 text-center">
+        <div class="col-lg-9 col-12 px-0 d-flex">
+          <div class="feedback mt-4 pt-4 w-100 text-left border-top">
+            <a href='{{ printf "%s%s" "https://github.com/vmware-tanzu/tanzu-dev-portal/issues/new?assignees=&labels=bug&template=bug_report.md&title=" .Title }}'><i class="fab fa-github mr-2"></i><span class="align-top">Report an issue</span></a>
```

Comment:
```suggestion
            <a href='{{ printf "%s%s" "https://github.com/vmware-tanzu/tanzu-dev-portal/issues/new?assignees=&labels=bug&template=bug_report.md&title=" .Title }}'><i class="fab fa-github mr-2"></i><span class="align-top">Report an issue</span></a>
```

I think we might have to do this with each link, but we might have to do some explicit URL encoding on this. I thought it was automatic, but for example, [this video](https://deploy-preview-937--tanzu-dev-portal.netlify.app/developer/videos/spring-cloud-data-flow-creating-and-deploying-streams/) creates an issue with the title of just "Creating"

Reply:
Hey @BrianMMcClain , I swapped out the printf method with your method and it looks to resolve this issue as well. Ready to merge?